package.json

```diff
@@ -579,6 +579,7 @@
     "graphql": "^14.0.2",
     "iconv-lite": "0.4.23",
     "js-base64": "^2.4.9",
+    "lodash": "^4.17.11",
```

Comment:
I'm actually depending on `lodash.isequal` in #1169. I'll have to consolidate after this is merged so it isn't redundant 🙈